ifrs17/DataModel/DataStructure.ipynb

```diff
@@ -1559,9 +1559,17 @@
+                "\n",
                 "\n<br> <code>PremiumAllocation</code> : defines the weight of Premium to be included in the Experience Adjustement AoC Type of the Technical Margin and is valid only for Group of Insurance Contract with LiabilityType : Liability for Remaining Coverage.",
-                "\n<br> <code>CashFlowPeriodicity</code> : defines the periodicity of the provided cash flows, it is not a mandatory input. Supported values can be found [here](../Constants/Enums#cashflowperiodicity). When the column *CashFlowPeriodicity* is missing from the input file it is assumed a monthly periodicity (default value). When the *CashFlowPeriodicity* column is present, a valid value must be entered. ",
-                "\n<br> <code>InterpolationMethod</code> : defines the interpolation method to be applied to interpolate the cash flow to the monthly granularity, it is not a mandatory input. Supported values can be found [here](../Constants/Enums#interpolation-method). When the *InterpolationMethod* is not present in the input file or the *CashFlowPeriodicity* has the default value, then the default value for the *InterpolationMethod* is applied. The default value is Not Applicable. ",
+                "\n",
+                "\n<br> <code>CashFlowPeriodicity</code> : defines the periodicity of the provided cash flows, it is not a mandatory input column. Supported values can be found [here](../Constants/Enums#cashflowperiodicity). When the column *CashFlowPeriodicity* is missing from the input file it is assumed a monthly periodicity (default value). When the *CashFlowPeriodicity* column is present, a valid value must be entered. ",
+                "\n",
+                "\n<br> <code>InterpolationMethod</code> : defines the interpolation method to be applied to interpolate the cash flow to the monthly granularity, it is not a mandatory input column. Supported values can be found [here](../Constants/Enums#interpolation-method). When the *InterpolationMethod* is not present in the input file or the *CashFlowPeriodicity* has the default value, then the default value for the *InterpolationMethod* is applied. The default value is 'NotApplicable'.",
+                "\n",
+                "\n<br> <code>ReleasePattern</code> : defines the pattern of factors that will be used to defer and amortize any AmountType over the course of contract term. It is not a mandatory input.  This pattern is used, if no *ReleasePattern* is provided in the Cashflows (which is specific for combinations of AmountType, EstimateType and AocType). If provided, the ReleasePattern needs at least a column with 'ReleasePattern0' header. The 'ReleasePattern0' corresponds to the first value of the AnnualCohort which is defined in [GroupOfContracts](#group-of-contracts). Pattern for multiple years can be provided by adding columns 'ReleasePattern1', 'ReleasePattern2' etc. Even if the ReleasePattern header is present, the values for any given DataNode are still optional.",
```

Comment:
remove AocType. The only relevant value is CL,C

Reply:
Add that the periodicity of the cashflow is consider to match the value of CashflowPeriodicity also entered in the parameter

Reply:
modified the comment
removed AoCTope mention
Added a comment to AoC CL description